vita3k/gui/src/gui.cpp

```diff
@@ -105,7 +107,16 @@ static void init_style() {
 
 static void init_font(GuiState &gui, HostState &host) {
     ImGuiIO &io = ImGui::GetIO();
+
+#ifdef _WIN32
+    if (host.dpi_scale > 1) {
+        gui.monospaced_font = io.Fonts->AddFontFromFileTTF("C:\\Windows\\Fonts\\consola.ttf", 13.0f * host.dpi_scale * host.dpi_scale, NULL, io.Fonts->GetGlyphRangesJapanese()); // multiply by dpi_scale twice to correctly scale
```

Comment:
Constant string should be made a separate variable with name explaining the usage